cupy_backends/cuda/libs/cusparse.pyx

```diff
@@ -1638,47 +1639,51 @@ cpdef setStream(intptr_t handle, size_t stream):
     check_status(status)
 
 
-cpdef size_t getStream(intptr_t handle) except? 0:
+cpdef size_t getStream(intptr_t handle) except? -1:
```

Comment:
It's highly likely that `getStream` so it's better to use `-1` (max value of `size_t`) instead of `0`.